extensions/gc/GarbageCollector.js

```diff
@@ -138,6 +138,8 @@ class GarbageCollector extends EventEmitter {
             topic: this._gcConfig.topic,
             groupId: this._gcConfig.consumer.groupId,
             concurrency: this._gcConfig.consumer.concurrency,
+            maxQueued: this._gcConfig.consumer.maxQueued,
+            ordered: this._gcConfig.consumer.ordered,
```

Comment:
ordered is not a "config" parameter : it is an intrinsic parameter of the implementation of each backbeat consumer/extension